### ISSUE TITLE
[feature request] Support local paths in package.json.

### ISSUE THREAD

**Issue by trusktr** [alexanderGugel/ied#46]
Like [this](https://docs.npmjs.com/files/package.json#local-paths).

I've got a project with an auto-generated package.json that looks like this:

```js
{
    "name": "rocket_module__root",
    "description": "root package",
    "version": "0.0.0",
    "license": "WTFPL",
    "repository": {
        "type": "git",
        "url": "git://github.com/meteor-rocket/module.git"
    },
    "dependencies": {
        "webpack": "^1.10.5",
        "rocket_module__rocket_module": "file:./packages/rocket_module__rocket_module",
        "rocket_module___app": "file:./packages/rocket_module___app"
    }
}
```

nom currently throws a 404 error about "rocket_module__rocket_module" not being in npmjs.com.

By the way, nom's speed is awesome.

**Comment by derhuerst** [comment]
Why not move/symlink the dependencies into `node_modules`? Then, no additional complexity for parsing local file paths would be needed.

**Comment by trusktr** [comment]
@derhuerst Will that also install the dependencies of those moved/symlinked packages?

**Comment by rstacruz** [comment]
> Why not move/symlink the dependencies into node_modules? 

This is not an npm-compatible way, unfortunately. I can't remember exactly why, but this caused many problems for me in deploying node.js apps.

**Comment by timdp** [comment]
+1

**Comment by mgcrea** [comment]
Landed in e8ff294. Please test against master to check if all your use cases are properly handled. Thanks!
